### ISSUE TITLE
not able to run the example

### ISSUE THREAD

**Issue by deepak1675** [nmrugg/stockfish.js#13]
engile is shows loading

**Comment by ajm2017** [comment]
Getting the same error - the index.html example doesn't seem to get past Engine: loading...

**Comment by nmrugg** [comment]
Try it now.

**Comment by ajm2017** [comment]
Works! Ty! I was actually interesting in running/timing a perft to compare this to a Javascript version of my own engine. Stockfish.JS is still remarkably fast even after porting! Well done!!

For Perft 2/5, on my machine, Intel Core 2, 2.00GHz, it returns 193690690 in 23.5s, roughly. My engine is about 64s and doesn't use magic bitboards. I've implemented a 32-bit version of magics in the past, but I couldn't get it to work faster than mailbox, so I reverted to the good ol' days. To get 64s, I think I've spent about equal time agonizing over JIT friendly code as I have in data structure and algorithmic design.

Did you convert all the bitboard logic Stockfish to 32bit integers? Or, are you using BigInt? Or, something else?
